test/suites/zombie_tanssi_relay_eth_bridge/test_zombie_tanssi_relay_eth_bridge.ts

```diff
@@ -491,12 +503,23 @@ describeSuite({
 
                 // In new era's first session at least one block need to be produced by the operator
                 const blocksPerSession = 10;
+                const countedSet = new Set();
                 for (let i = 0; i < 3 * blocksPerSession; ++i) {
                     const latestBlockHash = await relayApi.rpc.chain.getBlockHash();
                     const author = (await relayApi.derive.chain.getHeader(latestBlockHash)).author;
-                    if (author?.toString() === operatorAccount.address) {
-                        return;
+                    if (author) {
+                        if (
+                            [operatorAccount.address, operatorAccount2.address, operatorAccount3.address].includes(
+                                author.toString()
+                            )
+                        ) {
+                            countedSet.add(author.toString());
```

Comment:
This could be cleaner if you initialize the set to the 3 operators and then remove new authors from that set until its size is 0. This way when the test fails we can see which operator failed to produce a block.